package.json

```diff
@@ -90,9 +90,12 @@
     "rollup-plugin-node-resolve": "^5.2.0",
     "tslint": "^6.1.1",
     "tslint-config-prettier": "^1.18.0",
-    "typescript": "^4.0.3"
+    "typescript": "^4.4.3"
   },
   "engines": {
     "node": "6.* || 8.* || >= 10.*"
+  },
+  "resolutions": {
+    "@definitelytyped/typescript-versions": "0.0.89"
```

Comment:
This was necessary to avoid the yarn error:

```
Pattern ["@definitelytyped/typescript-versions@latest"] is trying to unpack in the same destination "/Users/ianvs/Library/Caches/Yarn/v6/npm-@definitelytyped-typescript-versions-0.0.89-815ba4ec6e2e0486e0d192ba19d7359cd9610f38-integrity/node_modules/@definitelytyped/typescript-versions" as pattern ["@definitelytyped/typescript-versions@^0.0.89","@definitelytyped/typescript-versions@^0.0.89","@definitelytyped/typescript-versions@^0.0.89"]. This could result in non-deterministic behavior, skipping.
```